modules/nixos.nix

```diff
@@ -160,7 +161,7 @@ in {
           # Constructed rule string that consists of the type, target, and source
           # of a tmpfile. Files with 'null' sources are filtered before the rule
           # is constructed.
-          ruleString = [mode file.target "- - - -" file.source];
+          ruleString = [mode "'${file.target}' - - - -" file.source];
         in
           concatStringsSep " " ruleString
```

Comment:
I think this part could just be one string, concatStringsSep is not *actually* doing anything here. It was added in anticipation of *longer* strings, but `L` and L+` don't actually take any more arguments.

Does not need to be addressed in this PR, but might as well.

Reply:
Sure, I'll fix.